src/test/groovy/edu/oregonstate/mist/locations/frontend/MockUriInfo.groovy

```diff
@@ -1,50 +1,106 @@
 package edu.oregonstate.mist.locations.frontend
+
 import javax.ws.rs.core.UriInfo
 import javax.ws.rs.core.UriBuilder
 import javax.ws.rs.core.PathSegment
 import javax.ws.rs.core.MultivaluedMap
 import javax.ws.rs.core.MultivaluedHashMap
 
 class MockUriInfo implements UriInfo {
```

Comment:
@magical good catch. I think we can let this go for now. One of the things that I have mentioned to Jared in the past is that I'm not 100% sold with the locations-api being an API. At first it made sense, but the role of that codebase is just to query backends and output data that can be given to ElasticSearch. I think the locations backend lends itself better to be a something else than API. The logic behind the backend doesn't have to be executed via HTTP.